libs/blocks/aside/aside.js

```diff
@@ -73,38 +73,6 @@ function decorateMedia(el) {
   });
 }
 
-function decorateVideo(container) {
-  const link = container.querySelector('a[href*=".mp4"]');
```

Comment:
@elan-tbx Currently milo utils only treat fragment and franklin or slack uploaded videos as autoblock and hence this was added to make sure that if any link remains in background that matched a video format, it should be converted to a video tag.

images-tv.adobe.com BG applied on main - https://main--milo--adobecom.hlx.page/drafts/mathuria/aside/asidebackground 

images-tv.adobe.com BG failing on the PR branch - https://ebartholomew-aside-bg-breakpoints--milo--adobecom.hlx.page/drafts/mathuria/aside/asidebackground

In case if this needs to be removed the following line in utils.js decorateAutoBlock can be updated to handle all videos in place of only uploaded videos 
`if (key === 'video' && !a.textContent.match('media_.*.mp4')) `

Also the following line in video.js would need to be updated to add the full href in source pathname if the href domain does not match the current domain so that it does not convert the external video links (for instance from images-tv.adobe.com) into relative paths -
<source src=".${pathname}" type="video/mp4" />

Reply:
@aishwaryamathuria thanks for the context!

@chrischrischris what do you think would be the best way to proceed here? I deferred to the auto-block in this case to avoid the race condition we were seeing, but not sure if we want to open up video.js to non-slack uploaded videos.

Reply:
Aside and Marquee are the only 2 blocks that support .mp4 background videos.  We don't want authors using .mp4 except for these 2 use cases.  We may be able to consolidate the code into decorate.js however I'm not sure if there are minor differences between the 2 block bg videos.

Reply:
@chrischrischris That makes sense. In that case, how would your recommend avoiding the video.js race condition once I bring back the function @aishwaryamathuria is talking about here? Is there a way I can opt out of auto-blocking with the background specifically?

Reply:
I just read through #1242 -- think I understand now. Thanks all.